temporal-sdk/src/main/java/io/temporal/worker/WorkerOptions.java

```diff
@@ -319,6 +323,33 @@ public Builder setDisableEagerExecution(boolean disableEagerExecution) {
       return this;
     }
 
+    /**
+     * Opts the worker in to the Build-ID-based versioning feature. This ensures that the worker
+     * will only recieve tasks which it is compatible with. For more information see: TODO: Doc link
+     *
+     * <p>Defaults to false
+     *
+     * <p>This is an EXPERIMENTAL API.
```

Comment:
Can we use the annotation for this `@Experimental`

Reply:
I'd need to upgrade the annotation, as it wasn't working in some spots for whatever reason, but seems like I can make it work there without much effort.

Reply:
Huh, I guess that works now. Weird.